egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -7855,372 +7855,113 @@
         }
       ]
     },
+
     {
-      "name": "NURT_DASHBOARD",
```

Comment:
WHY NURT_DAHSOBARD is changed.

Please add the new one for PGR.. NURT Dashabord is for landing page.